src/DBFTPlugin/Messages/ConsensusMessage.cs

```diff
@@ -36,6 +36,8 @@ public virtual void Deserialize(BinaryReader reader)
         public static ConsensusMessage DeserializeFrom(byte[] data)
         {
             ConsensusMessageType type = (ConsensusMessageType)data[0];
+            if (!Enum.IsDefined(typeof(ConsensusMessageType), type))
+                throw new ArgumentOutOfRangeException(nameof(type));
             Type t = typeof(ConsensusMessage);
             t = t.Assembly.GetType($"{t.Namespace}.{type}", false);
             if (t is null) throw new FormatException();
```

Comment:
If we don't modify it, then it will throw an exception here, right?

Reply:
> If we don't modify it, then it will throw an exception here, right?

Yes, t is null and throw an exception. Maybe his point is to make the exception information more clear. :joy:

Reply:
I think so, but https://github.com/neo-project/neo-modules/pull/684#discussion_r790356491

Reply:
I don't know, but i did not see any sign of any exception here.

<img width="1351" alt="image" src="https://user-images.githubusercontent.com/10189511/151015685-0bed7fae-48e8-46b0-8f69-a01bc86d50e3.png">

Reply:
Here in `ConsensusService`:

```C#
   ConsensusMessage message;
            try
            {
                message = context.GetMessage(payload);
            }
            catch (FormatException)
            {
                return;
            }
            catch (IOException)
            {
                return;
            }
```

The exception is ignored directly.

Reply:
I think that it should be logged, but type is working as a switch